packages/expo-cli/src/commands/build/ios/appleApi/distributionCert.js

```diff
@@ -26,7 +26,9 @@ const createManager = ({ appleId, appleIdPassword, team }) => ({
     } catch (err) {
       const resultString = get(err, 'rawDump.resultString');
       if (resultString && resultString.match(/Maximum number of certificates generated/)) {
-        log.error(APPLE_DIST_CERTS_TOO_MANY_GENERATED_ERROR);
+        let error = new Error(APPLE_DIST_CERTS_TOO_MANY_GENERATED_ERROR);
```

Comment:
I prefer to use `const` for things that don't change.

Reply:
Ok!